templates/personal-settings.php

```diff
@@ -93,5 +93,5 @@
 	<p class="settings-hint"><?php p($l->t('Use this link to connect your apps and desktop client to this server:')); ?></p>
 
 	<input type="text" readonly="readonly" id="endpoint-url" value="<?php echo p($_['url']); ?>" />
-	<a class="button clipboardButton icon-clippy" data-clipboard-target="#endpoint-url"><?php p($l->t('Copy link')); ?></a>
+	<a class="button clipboardButton icon-clippy" data-clipboard-target="#endpoint-url" title="<?php p($l->t('Copy to clipboard')) ?>"><?php p($l->t('Copy link')); ?></a>
```

Comment:
Since the button has text, the title should not be needed?

Reply:
Not sure that makes lots of sense but this is just a replacement for custom tooltip (rewritten).

Reply:
I see that, I still think it's weird++ and I would drop it.